attractions/checkbox/checkbox.svelte

```diff
@@ -1,18 +1,58 @@
 <script>
+  /**
+   * @event {{ value: string; checked: boolean; nativeEvent: Event }} change
+   */
   import { createEventDispatcher } from 'svelte';
   import classes from '../utils/classes.js';
 
   let _class = null;
   export { _class as class };
+  /**
+   * A class string to assign to the `<input>` element.
+   * @type {string}
+   */
   export let inputClass = null;
+  /**
+   * A class string to add to the selector box element.
+   * @type {string}
+   */
   export let selectorClass = null;
+  /**
+   * A CSS style string to assign to the selector box element.
+   * Can be used to make the checkbox represent a color (in conjunction with the `getColorPickerStyles` utility).
+   * @type {string}
+   */
   export let selectorStyle = null;
 
+  /**
+   * Whether the checkbox is checked or not.
+   * @type {boolean}
+   */
   export let checked = false;
+  /**
+   * The value assigned to the checkbox input. Check [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#Value) for more information.
+   * @type {string}
```

Comment:
I thought Svelte allowed this only with `<select>`s. Wouldn't the object be stringified to `[object Object]`?

Reply:
It would, but it doesn't matter, everything still works, it's just that the DOM representation is meaningless.
But I think we shouldn't rely too much on that, and demand them all to be strings.

Reply:
No, I don't think it is just a representation. I remember facing many warning about having duplicate `id`s (or something else, not quite sure): `[object Object]`. They would probably have the same value. This needs testing

Reply:
I will leave it as `string` for consistency as you said in another comment